CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 - Allow `define_metric("x", step_metric="x")` when using core (@timoffex in https://github.com/wandb/wandb/pull/8107)
 - Correctly upload empty files when using core (@timoffex in https://github.com/wandb/wandb/pull/8109)
+- `Artifact.download()` now respects `skip_cache=True` when using wandb-core by @moredatarequired in https://github.com/wandb/wandb/pull/7640
```

Comment:
Is this correct? In `downloader.go`, `SkipCache` is still documented as "Currently unused".